snuba/datasets/factory.py

```diff
@@ -8,7 +8,7 @@
 DATASETS_IMPL: MutableMapping[str, Dataset] = {}
 DATASETS_NAME_LOOKUP: MutableMapping[Dataset, str] = {}
 
-DEV_DATASET_NAMES: Set[str] = set()
+DEV_DATASET_NAMES: Set[str] = set("profiles")
```

Comment:
Should be `{"profiles"}` or `set(["profiles"])`

Reply:
Indeed, what I wrote ends up a little... weird.